AlgoRhythm/README.md

```diff
@@ -32,7 +32,74 @@ of the algorithmic principles and how to solve a given problem in the most optim
 <h5>Note:</h5> If you are able to solve the problems during challenge using any other technique apart from the ones that mentioned above, it's 
 perfectly fine.
 
----------
+---
+
+### Resources :
+
+1. Basics :
+* Arrays
+* Strings
+* Stacks
+* Queues
+* Linked Lists
+* [Binary-Search](https://www.topcoder.com/community/competitive-programming/tutorials/binary-search)
+* Sort
+* Bit Manipulation
+* Disjoint Set union
+
+2. Questions and Resources : 
+* [CodeMonk](https://www.hackerearth.com/practice/codemonk/)
+
+3. Game Theory :
+* Nim
+* Sprague-Grundy Theorem
+* First try this [**question**](https://www.hackerrank.com/challenges/game-of-stones-1/problem) without reading any theory.
+
+4. Theory :
```

Comment:
Do not number this line. It's sub-topic

Reply:
got it theory and questions are sub-topic